packages/labextension/src/index.ts

```diff
@@ -26,9 +27,117 @@ import {
 } from './validator';
 
 const CONDAENVID = '@mamba-org/gator-lab:plugin';
+const CONDASTOREENVID = '@condastore/gator-lab:plugin';
```

Comment:
Let's stick with the convention of using the package name:

```suggestion
const CONDASTOREENVID = '@mamba-org/conda-store:plugin';
```